src/reporter.ts

```diff
@@ -43,13 +43,16 @@ function getMarkdownFormat(
     table: {
       head: ['Login', 'Emails', 'Orgs', 'Membership', 'Created At'],
       body: [
-        ...allMembers.map(item => [
-          item.login,
-          item.emails.join(','),
-          item.orgs.join(','),
-          item.type.toString(),
-          item.createdAt
-        ])
+        ...allMembers
+          .sort((a, b) => a.login.localeCompare(b.login))
+          .map(item => [
+            item.login,
+            // Make sure the email only appears once
+            Array.from(new Set(item.emails)).join(','),
```

Comment:
```suggestion
            [...new Set(item.emails)].join(','),
```

Might be the easier syntax.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#remove_duplicate_elements_from_the_array